test/conformance/service_test.go

```diff
@@ -440,26 +436,42 @@ func TestReleaseService(t *testing.T) {
 	if err := validateAnnotations(objects); err != nil {
 		t.Errorf("Service annotations are incorrect: %v", err)
 	}
-	revisions := []string{names.Revision}
+	firstRevision := names.Revision
 
 	// 1. One Revision Specified, current == latest.
 	t.Log("1. Updating Service to ReleaseType using lastCreatedRevision")
-	objects.Service, err = test.PatchReleaseService(t, clients, objects.Service, revisions, 0)
+	objects.Service, err = test.UpdateServiceRouteSpec(t, clients, names, v1alpha1.RouteSpec{
+		Traffic: []v1alpha1.TrafficTarget{{
+			TrafficTarget: v1beta1.TrafficTarget{
+				Tag:          "current",
```

Comment:
```suggestion
				Tag:          v1alpha1.CurrentTrafficTarget,
```

Reply:
🤔  The client is now in total control of these, so I'm inclined to nack.  Is there a strong reason to use the constant instead?

Reply:
Your checks below for the resulting traffic uses the constants. Setting it as a string up here and checking for a constant value below is misleading and leaves us straddling the old behavior and the new client convention.

It might be nice to instead distance this test from the client convention by moving away from the constants as well as the names "current", "candidate", and "latest". That makes it clear that the behavior here is being defined by the user/client and not the system.